documentation/modules/managing/proc-drain-cleaner-deploying.adoc

```diff
@@ -86,7 +86,7 @@ spec:
 ----
 +
 Reducing the maximum pod disruption budget to zero prevents Kubernetes from automatically evicting the pods in case of voluntary disruptions,
-so pods must be evicted by the Strimzi Drain Cleaner.
+leaving the floor for pods to be evicted by the Strimzi Drain Cleaner without conflict.
```

Comment:
```suggestion
leaving the Strimzi Drain Cleaner free to evict pods without conflict.
```

Reply:
It is the joint work of Drain Cleaner, Cluster Operator and K8s Scheduler to actually move pods away from the cordoned node.

Reply:
Yeah, I think Federico has a point. Maybe something like

```suggestion
leaving the Strimzi Drain Cleaner and Strimzi Cluster Operator to roll the pod which will be scheduled by Kubernetes on a different worker node.
```